src/carousel.coffee

```diff
@@ -81,6 +81,11 @@ Ember.Widgets.CarouselComponent = Ember.Component.extend
         $active.removeClass(['active', direction].join(' '))
         @set 'sliding', no
 
+        # Send an action out to signal that the sliding transition has ended
+        # It can is useful to avoid having observers in any parentView having
+        # this component.
+        @send 'transitionEnded'
```

Comment:
I agree, but I don't think documenting ember concepts around individual actions within a component library is necessary. Documentation about ember concepts should be part of the ember documentation, documentation about ember widgets should be part of this repo.